internal/store/postgres/postgres.go

```diff
@@ -20,25 +22,65 @@ func New(c Config) (*gorm.DB, error) {
 		c.Password,
 	)
 
-	db, err := gorm.Open(gormpg.Open(dsn), &gorm.Config{Logger: logger.Default.LogMode(getLogLevelFromString(c.LogLevel))})
+	db, err := gorm.Open(gormpg.Open(dsn), &gorm.Config{Logger: gormlogger.Default.LogMode(getLogLevelFromString(logger.Level()))})
 	if err != nil {
 		panic(err)
 	}
 
 	return db, err
 }
 
-func getLogLevelFromString(level string) logger.LogLevel {
+func getLogLevelFromString(level string) gormlogger.LogLevel {
 	switch level {
 	case "silent":
-		return logger.Silent
+		return gormlogger.Silent
 	case "error":
-		return logger.Error
+		return gormlogger.Error
 	case "warn":
-		return logger.Warn
+		return gormlogger.Warn
 	case "info":
-		return logger.Info
+		return gormlogger.Info
 	default:
-		return logger.Info
+		return gormlogger.Info
 	}
 }
+
+func Migrate(logger log.Logger, db *gorm.DB) error {
+	logger.Info("migrating alert table...")
+	err := db.AutoMigrate(&model.Alert{})
+	if err != nil {
+		return err
+	}
+	logger.Info("migrating namespace table...")
+	err = db.AutoMigrate(&model.Namespace{})
+	if err != nil {
+		return err
+	}
+	logger.Info("migrating provider table...")
+	err = db.AutoMigrate(&model.Provider{})
+	if err != nil {
+		return err
+	}
+	logger.Info("migrating receiver table...")
+	err = db.AutoMigrate(&model.Receiver{})
+	if err != nil {
+		return err
+	}
+	logger.Info("migrating rule table...")
+	err = db.AutoMigrate(&model.Rule{})
+	if err != nil {
+		return err
+	}
+	logger.Info("migrating subscription table...")
+	err = db.AutoMigrate(&model.Subscription{})
+	if err != nil {
+		return err
+	}
+	logger.Info("migrating template table...")
+	err = db.AutoMigrate(&model.Template{})
```

Comment:
we can pass multiple models as params in the AutoMigrate func -> `db.AutoMigrate(&model.Alert{}, &model.Namespace{}, ...)`, would be simpler that way

Reply:
oh nice thanks @rahmatrhd !